content/en/docs/concepts/data-sources.md

```diff
@@ -3,147 +3,14 @@ title: Data sources
 weight: 30
 ---
 
-OpenTelemetry supports multiple data sources as defined below. More data sources
-may be added in the future.
+In OpenTelemetry, a Data Source refers to the type of data that supported by the
+specification. It currently supports the Data Sources below, and more may be
+added in the future.
 
-## Traces
+- [Traces](/docs/concepts/otel-concepts/#tracing-in-opentelemetry)
```

Comment:
Do we follow this specific order for a reason, if not, consider alphabetical order.

Reply:
I wanted to put Traces & Spans in the beginning since they're more mature, followed by Metrics, and then Logs, as less mature. I left Baggage at the end since it's (at least to me) a little more esoteric?